bindings/pyroot/pythonizations/test/rbatchgenerator_completeness.py

```diff
@@ -0,0 +1,229 @@
+import unittest
+import os
+import ROOT
+
+class RBatchGeneratorMultipleFiles(unittest.TestCase):
+
+    file_name1 = "first_half.root"
+    file_name2 = "second_half.root"
+    tree_name = "mytree"
+
+    # Helpers
+    def create_10_entries_file(self):
+        df2 = ROOT.RDataFrame(10)\
+            .Define("b1", "(int) rdfentry_")\
+            .Define("b2", "(double) b1*b1")\
+            .Snapshot(self.tree_name, self.file_name1)
+    
+    # def create_20_entries_file(self):
+    #     df1 = ROOT.RDataFrame(20)\
+    #         .Define("b1", "(int) rdfentry_ + 10")\
+    #         .Define("b2", "(double) rdfentry_ * rdfentry_")\
+    #         .Snapshot(self.tree_name, self.file_name2)
+        
+        #print(df1.Describe())
+        #print(df2.Describe())
+
+        #df2.Display("",).Print()
+
+
+    def teardown_file(self, file):
+        os.remove(file)
```

Comment:
This function should not be there. setup and teardown routines for Python unittests are handled via `setup*` and `tearDown*` functions. See the documentation at https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass

Reply:
Will improve, thanks!